lib/cucumber/listener/pretty_formatter.js

```diff
@@ -85,11 +90,12 @@ var PrettyFormatter = function(options) {
     var source = self.applyColor(stepResult, step.getKeyword() + (step.getName() || ''));
     source = self._pad(source, self._getCurrentMaxStepLength() + 10);
 
-    if (step.hasUri()) {
-      source += color.format('comment', "# " + step.getUri().replace(process.cwd(),'').slice(1) + ":" + step.getLine());
-    }
 
+    if (options.showSource && step.hasUri()) {
```

Comment:
Should probably put the whitespace back to what is was. 1 line above and 1 below this block instead of 2 above and 0 below